libguard/guard_interface.cpp

```diff
@@ -159,6 +159,7 @@ GuardRecord create(const EntityPath& entityPath, uint32_t eId, uint8_t eType)
                 offset = lastPos * sizeOfGuard;
                 existGuard.errType = eType;
                 existGuard.recordId = htobe32(lastPos + 1);
```

Comment:
Looks like, you are overriding the existing Reconfig record if so, why you are increasing the record id? It should be the same, right?

Reply:
Nope it can't be same as we treat it as new record, even though it was logged as reconfig type in previous case.

Reply:
What do you mean by "we treat it as new record"?

Let say,

```
 1 Reconfig core0
 2 Manual   core1
```

If we are overriding the Reconfig record by the manual what is the expected output?